tests/e2e/developer_pages.py

```diff
@@ -465,3 +487,53 @@ def click_save_and_continue(self, collection_name: str) -> TasklistPage:
         task_list_page = TasklistPage(self.page, self.domain, self.grant_name, collection_name=collection_name)
         expect(task_list_page.heading).to_be_visible()
         return task_list_page
+
+
+class SubmissionsListPage(GrantDevelopersBasePage):
+    collection_name: str
+
+    def __init__(self, page: Page, domain: str, grant_name: str, collection_name: str) -> None:
+        super().__init__(
+            page,
+            domain,
+            grant_name=grant_name,
+            heading=page.get_by_role("heading", name=f"{collection_name} Submissions"),
+        )
+        self.collection_name = collection_name
+
+    def click_on_collection(self, collection_reference: str) -> ViewCollectionResponsesPage:
+        self.page.get_by_role("link", name=collection_reference).click()
+        view_collection_page = ViewCollectionResponsesPage(
+            self.page, self.domain, self.grant_name, collection_id=collection_reference
+        )
+        expect(view_collection_page.heading).to_be_visible()
+        return view_collection_page
+
+
+class ViewCollectionResponsesPage(GrantDevelopersBasePage):
+    collection_id: str
```

Comment:
Still good to call this `collection_reference` I think but but can see that its now used in grabbing the heading!